openmls/src/ciphersuite/mac.rs

```diff
@@ -1,17 +1,44 @@
 use super::*;
 
+/// A wrapper to contain MAC values that will be zeroed on drop.
+#[derive(
+    Debug,
+    Clone,
+    Serialize,
+    Deserialize,
+    TlsDeserialize,
+    TlsSerialize,
+    TlsSize,
+    zeroize::ZeroizeOnDrop,
+)]
+pub(crate) struct MacValue(Vec<u8>);
```

Comment:
MACs are generally considered public information. Is there a reason you want to derive ZeroizeOnDrop for them?

Reply:
I didn't know. I thought it was secret as it can be constructed from a secret.

Reply:
Fair point. In fact I just realized that we're being a bit sloppy by computing a MAC by calling HKDF extract. That works, but only as long as the HPKE ciphersuite uses an HKDF as its key derivation function. I filed #1375 to fix this (although it's functionally correct for now).

That being said, MACs are constructed in a similar way that ciphertexts are constructed from an encryption key, so I think we can do away with zeroize here.